src/js/letters/reducers/index.js

```diff
@@ -123,6 +133,41 @@ function letters(state = initialState, action) {
       return _.set('address', action.address, newState);
     }
     // Add SAVE_ADDRESS_FAILURE
+    case GET_ADDRESS_COUNTRIES_SUCCESS: {
+      let countriesAvailable = true;
+      const countryList = action.countries.data.attributes.countries;
+
+      // Log error if the countries response is not what we expect
+      if (!_.isArray(countryList) || countryList.length === 0) {
```

Comment:
@cvalarida I took your suggestion of logging a Sentry error if the response for countries and states isn't what we expect. Do you mind taking another look at this to make sure it makes sense? I'm basically checking if either the response isn't an array or if it's an empty array, and then both logging a Sentry error and making the countries/states unavailable.

Reply:
Looks good! I'd just like to reduce lodash dependency, so my only nitpick here is that I'd rather see `Array.isArray(countryList)`.

If you're tired of changing things, though, that's totally fine. 😛